src/components/Dialog/index.test.tsx

```diff
@@ -32,27 +32,26 @@ describe('Dialog', () => {
     expect(screen.getByText('subTitle')).toBeInTheDocument();
   });
 
-  it('tests that onDismiss gets fired onClick of close', () => {
+  it('Tests that onDismiss gets fired correctly', () => {
     render(dialog);
-
     expect(screen.getByText('content')).toBeInTheDocument();
     fireEvent.click(screen.getByLabelText('Close'));
     expect(screen.queryByText('content')).not.toBeInTheDocument();
-    expect(dismiss).toHaveBeenCalledTimes(1);
-  });
 
-  it('tests that pressing Esc anywhere in dialog should trigger a close', () => {
     render(dialog);
-
     fireEvent.keyDown(screen.getByTestId('dialog'), { key: 'Escape', code: 27 });
-    expect(dismiss).toHaveBeenCalledTimes(1);
+    expect(dismiss).toHaveBeenCalledTimes(2);
```

Comment:
whoops 👍 this should have definitely been 2 before (one of the benefits of consuming a library, it just fixes this stuff).